warehouse/upload.go

```diff
@@ -1758,7 +1759,11 @@ func (job *UploadJobT) createLoadFiles(generateAll bool) (startLoadFileID int64,
 			}
 
 			if misc.ContainsString(warehouseutils.TimeWindowDestinations, job.warehouse.Type) {
-				payload.LoadFilePrefix = stagingFile.TimeWindow.Format(warehouseutils.DatalakeTimeWindowFormat)
+				if job.warehouse.Type == warehouseutils.GCS_DATALAKE {
```

Comment:
minor: We can move this to a separate files and write a Table Driven test for these kind of things going forward.